teuthology/provision/pelagos.py

```diff
@@ -0,0 +1,156 @@
+
+import logging
+import requests
+import time
+from urllib2 import  HTTPError
```

Comment:
Can we use teuthology.util.compat to import urllib related things

Reply:
fixed